t/16-schema-upgrade-html.t

```diff
@@ -147,27 +157,32 @@ $out = html_formatter_context {
 } test_request();
 
 filter_js_src($out);
+my $mode = find_application_mode($out);
 
-my @expected = split (/\n/, q{<!-- prettier-disable -->
+my @expected = split (/\n/, qq{<!-- prettier-disable -->
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
     <title></title>
     <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
-    <link rel="stylesheet" href="js/dojo/resources/dojo.css" type="text/css" />
-    <link rel="stylesheet" href="js/dijit/themes/claro/claro.css" type="text/css" />
-    <link rel="stylesheet" href="css/ledgersmb.css" type="text/css" />
-    <link rel="stylesheet" href="css/setup.css" type="text/css" />
+    <link href="js/css/claro.css" rel="stylesheet">
+    <link href="js/css/ledgersmb.css" rel="stylesheet">
+    <link href="js/css/setup.css" rel="stylesheet">
```

Comment:
why is `js/css/` now a directory? it's ugly to put `css/` in the `js/` namespace, don't you think?

Reply:
All CSS are now compiled.

Reply:
Our UI/js is dist in webpack world, i.e. where all compiled artefacts are put. Except for most HTML files, that's the case now.

Reply:
ok. I still don't like the idea that all our assets are in `js/` (including css and images). Maybe we need to restructure the `UI/` directory to separate out the static assets from the UI templates?

Reply:
Sources remained where they were, only compiled artefacts go there. Why should we care about their internals?

Webpack default to `/dist` (UI/js, for us, to prevent Web proxy modifications) and packs everything compiled there.

Reply:
Hmm. Maybe we should set up a `dist/` outside of the UI directory?

Reply:
or at least rename `js/` to `dist/` ?

Reply:
It has to be inside, to let the web proxy reach it and I kept it as UI/js for the same purpose. No problem to set it as /dist as soon as html get handled by webpack.

Reply:
ok.